lib/controllers/flight.py

```diff
@@ -5,7 +5,8 @@
 
 import jsonpickle
 
-from lib.models.rocket import Rocket
+from lib.models.rocket import Rocket, RocketOptions
+from lib.models.motor import Motor, MotorEngines, MotorKinds
 from lib.models.flight import Flight
 from lib.models.environment import Env
 from lib.views.flight import FlightSummary, SurfaceWindConditions, OutOfRailConditions, BurnoutConditions, ApogeeConditions, MaximumValues, InitialConditions, NumericalIntegrationSettings, ImpactConditions, EventsRegistered, LaunchRailConditions, FlightData, FlightCreated, FlightUpdated, FlightDeleted, FlightPickle
```

Comment:
This is an example of too long line. The code would benefit from a linter like black

https://black.readthedocs.io/en/stable/

Reply:
Nice catch, I forgot about the imports. I think there is a flag for that in pylint as well, I will check.